pkg/finder/finder.go

```diff
@@ -26,9 +28,10 @@ import (
 )
 
 const (
-	defaultTimeout   = 5 * time.Second
-	ns               = "resource-topology-exporter"
-	deviceNamePrefix = "EXAMPLECOMDEVICE"
+	defaultTimeout     = 5 * time.Second
+	ns                 = "resource-topology-exporter"
+	cgroupCpuSetPrefix = "/host-sys/fs/cgroup/cpuset/"
```

Comment:
I'd not use an absolute path here, because doing so leaks in the pod definition. In other word, if the host sysfs is mounted somewhere else than on `/host-sys`, this code unnecessarily breaks. We should get the sysfs mount point as parameter

Reply:
Ah yeah, that makes sense, will update this!